src/iOS/Xamarin.iOS/Samples/Augmented Reality/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cs

```diff
@@ -0,0 +1,192 @@
+            _arKitStatusLabel.TextAlignment = UITextAlignment.Center;
+            _arKitStatusLabel.TextColor = UIColor.Black;
+            _arKitStatusLabel.BackgroundColor = UIColor.FromWhiteAlpha(1.0f, 0.6f);
+            _arKitStatusLabel.Text = "Setting up ARKit";
+
+            // Add the views.
+            View.AddSubviews(_arSceneView, _arKitStatusLabel);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
+                _arSceneView.TopAnchor.ConstraintEqualTo(View.TopAnchor),
+                _arSceneView.BottomAnchor.ConstraintEqualTo(View.BottomAnchor),
+                _arSceneView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _arSceneView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                _arKitStatusLabel.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                _arKitStatusLabel.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _arKitStatusLabel.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                _arKitStatusLabel.HeightAnchor.ConstraintEqualTo(40)
+            });
+
+            // Listen for tracking status changes and provide feedback to the user.
+            _arSceneView.ARSCNViewCameraDidChangeTrackingState += CameraTrackingStateChanged;
+        }
+
+        private async void Initialize()
+        {
+            // Create the scene with a basemap.
+            Scene flyoverScene = new Scene(Basemap.CreateImagery());
+
+            // Create the integrated mesh layer and add it to the scene.
+            IntegratedMeshLayer meshLayer = new IntegratedMeshLayer(new System.Uri("https://www.arcgis.com/home/item.html?id=dbc72b3ebb024c848d89a42fe6387a1b"));
+            flyoverScene.OperationalLayers.Add(meshLayer);
+
+            try
+            {
+                // Wait for the layer to load so that extent is available.
+                await meshLayer.LoadAsync();
+
+                // Start with the camera at the center of the mesh layer.
+                Envelope layerExtent = meshLayer.FullExtent;
+                Camera originCamera = new Camera(layerExtent.GetCenter().Y, layerExtent.GetCenter().X, 600, 0, 90, 0);
+                _arSceneView.OriginCamera = originCamera;
+
+                // set the translation factor to enable rapid movement through the scene.
+                _arSceneView.TranslationFactor = 1000;
+
+                // Enable atmosphere and space effects for a more immersive experience.
+                _arSceneView.SpaceEffect = SpaceEffect.Stars;
+                _arSceneView.AtmosphereEffect = AtmosphereEffect.Realistic;
+
+                // Display the scene.
+                await flyoverScene.LoadAsync();
+                _arSceneView.Scene = flyoverScene;
+            }
+            catch (Exception ex)
+            {
+                ShowMessage("Failed to start AR", "Error starting");
```

Comment:
new UIAlertView("Error", "Failed to start AR", (IUIAlertViewDelegate) null, "OK", null).Show();